### ISSUE TITLE
`wwctl overlay show -r ...` tries to perform `fmt` operations on template before/after rendering

### ISSUE THREAD

**Issue by griznog** [warewulf/warewulf#1363]
### Steps to reproduce

Create a template or just try to use `%` in a template anywhere inside or outside of `{{ }}` or in a tag value.  

### Error message

There's no error from `wwctl` but the output of `wwctl overlay show -r ...` will have tried to do some `fmt` operation on the template or result, which breaks processing %. 

`wwctl overlay build ...` handles `%` properly, without any modification. 

### Information on your system

Running latest `main` from github on Rocky Linux 8 and 9. 

### General information

- [X] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have captured and reported relevant error messages and logs

**Comment by griznog** [comment]
I think I understand what's happening here, but am not sure how to fix it. The rendered template is displayed using `wwlog.INFO`

https://github.com/warewulf/warewulf/blob/09c17551e869deb7403bf277905c397870026ae6/internal/app/wwctl/overlay/show/main.go#L116

The path through `wwlog` then manipulates the output before dumping it out. If I replace this line with 

```
fmt.Print(outBuffer.String())
```

Then it works correctly and as expected. `wwlog` seems like a deep rabbit hole, is there any reason this can't/shouldn't just be a simple dump to `stdout`?

**Comment by anderbubble** [comment]
I agree that this should just be `fmt.Print` or similar; but that's an argument from the past that I won't re-litigate here. ^_^

For now, it looks like this was already solved in a different place in the same file like this:

```
wwlog.Info("%s", outBuffer.String())
```

So I'll test a patch like that here, and submit a PR.

**Comment by anderbubble** [comment]
@griznog give the attached PR a try and let me know what you think. (Approve if it fixes it for you, if you don't mind, and I'll get it merged and backported for v4.5.7.)
